configurations/sync_pool_wusc_to_text_it.py

```diff
@@ -0,0 +1,67 @@
+    ),
+    uuid_table=UUIDTableClientConfiguration(
+        credentials_file_url="gs://avf-credentials/avf-id-infrastructure-firebase-adminsdk-6xps8-b9173f2bfd.json",
+        table_name="avf-global-urn-to-participant-uuid",
+        uuid_prefix="avf-participant-uuid-"
+    ),
+    operations_dashboard=OperationsDashboardConfiguration(
+        credentials_file_url="gs://avf-credentials/avf-dashboards-firebase-adminsdk-gvecb-ef772e79b6.json",
+    ),
+    rapid_pro_target=RapidProTarget(
+        rapid_pro=RapidProClientConfiguration(
+            domain="textit.com",
+            token_file_url="gs://avf-credentials/wusc-leap-kalobeyei-textit-token.txt"
+        ),
+        sync_config=EngagementDBToRapidProConfiguration(
+            normal_datasets=[
+                DatasetConfiguration(
+                    engagement_db_datasets=["kakuma_preffered_language"],
+                    rapid_pro_contact_field=ContactField(key="pool_wusc_preferred_language", label="pool_wusc_preferred_language")
+                ),
+                DatasetConfiguration(
+                    engagement_db_datasets=["kakuma_gender"],
+                    rapid_pro_contact_field=ContactField(key="pool_wusc_gender", label="pool wusc gender")
+                ),
+                DatasetConfiguration(
+                    engagement_db_datasets=["kakuma_location"],
+                    rapid_pro_contact_field=ContactField(key="pool_wusc_location", label="pool wusc location")
+                ),
+                DatasetConfiguration(
+                    engagement_db_datasets=["kakuma_nationality"],
+                    rapid_pro_contact_field=ContactField(key="pool_wusc_disabled", label="pool wusc nationality")
+                ),
+                DatasetConfiguration(
+                    engagement_db_datasets=["kakuma_age"],
+                    rapid_pro_contact_field=ContactField(key="pool_wusc_age", label="pool wusc age"),
+                ),
+                DatasetConfiguration(
+                    engagement_db_datasets=["kakuma_disabled"],
+                    rapid_pro_contact_field=ContactField(key="pool_wusc_disabled", label="pool wusc disabled")
+                ),
+            ],
+            consent_withdrawn_dataset=DatasetConfiguration(
+                engagement_db_datasets=["kakuma_age", "kakuma_gender", "kakuma_location", "kakuma_disabled",
+                                        "kakuma_preffered_age", "kakuma_nationality", "kakuma_old_rqa_datasets",
+                                        "leap_s04e01","leap_s04e02","leap_s04e03", "leap_s04e04",
+                                        "leap_s04e05", "leap_s04e06"],
+                rapid_pro_contact_field=ContactField(key="pool_wusc_consent_withdrawn",
+                                                     label="pool wusc consent withdrawn")
+            ),
+            write_mode=WriteModes.CONCATENATE_TEXTS,
+            allow_clearing_fields=False,
```

Comment:
I think this should be True for a sync back, so we can re-ask any questions that participants didn't give a relevant response to last time.